### PR TITLE
Add how-to guides

### PR DESCRIPTION
**Pull request by weiiwang01**
Applicable spec: <link>

### Overview

Add two how-to guides: one on setting up the NTS and another on configuring the time sources for the Chrony charm.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: docs/how-to/enable-nts.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# How to: Enable NTS (Network Time Security)
```
Comment (erinecon) [inline docs/how-to/enable-nts.md:1]: I would rephrase this title as `# How to enable Network Time Security`

At the very least, I recommend removing the colon from the title

File: docs/how-to/enable-nts.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## `tls-certificates` integration
+
+The Chrony charm can integrate with any charm that provides `tls-certificates` 
+integration, such as the [`lego`](https://charmhub.io/lego) or 
+[`self-signed-certificates`](https://charmhub.io/self-signed-certificates) 
+charms. 
+Refer to the respective `tls-certificates` provider charm's documentation 
+for setup instructions. 
+Once set up, you can integrate the `tls-certificates` provider charm with 
+the Chrony charm:
```
Comment (erinecon) [inline docs/how-to/enable-nts.md:23]: I think it would be good to indicate that you're showing an example below

File: docs/how-to/enable-nts.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+charm. This is especially useful if you want to generate a 
+[TLS certificate with an extended validity period for use with NTS clients lacking a real-time clock](https://chrony-project.org/faq.html#_using_nts). 
+
+The TLS certificate must be stored in [a Juju user secret](https://juju.is/docs/juju/manage-secrets#add-a-secret) 
+before being passed to the Chrony charm.
+
+```
+juju add-secret my-tls-cert cert#file=/path/to/fullchain.pem key#file=/path/to/key.pem
+juju grant-secret my-tls-cert chrony
+juju config chrony nts-certificates=secret:...
```
Comment (erinecon) [inline docs/how-to/enable-nts.md:42]: For my understanding: The `...` in the configuration command is a placeholder for whatever output you get from the `juju grant-secret` command?

File: docs/how-to/setup-time-sources.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# How to: Setup Time Sources
```
Comment (erinecon) [inline docs/how-to/setup-time-sources.md:1]: I would rephrase this title as `# How to set up time sources`

At the very least, I recommend removing the colon and splitting setup into two words

File: docs/how-to/setup-time-sources.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+Chrony alone cannot provide time; it requires another time source, either a 
+reference clock or another NTP server, to supply time to Chrony itself. 
+Currently, the Chrony charm supports only using another NTP server as the
```
Comment (erinecon) [inline docs/how-to/setup-time-sources.md:5]: Nitpick: I would flip the order of "supports" and "only", in other words:
`Currently, the Chrony charm only supports using another NTP...`

File: docs/how-to/setup-time-sources.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# How to: Setup Time Sources
+
+Chrony alone cannot provide time; it requires another time source, either a 
+reference clock or another NTP server, to supply time to Chrony itself. 
+Currently, the Chrony charm supports only using another NTP server as the 
+time source.
+
+## Use Another NTP Server
```
Comment (erinecon) [inline docs/how-to/setup-time-sources.md:8]: The [style guide](https://docs.ubuntu.com/styleguide/en/#headings) recommends using sentence case for headings, in other words:
`## Use another NTP server`

File: docs/how-to/setup-time-sources.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+The domain in the URL will be treated as a pool address for the NTP server.
+
+You can adjust the time source using 
+[pool options listed in the Chrony configuration](https://chrony-project.org/doc/4.6.1/chrony.conf.html). 
+These options are passed to the Chrony charm as URL parameters. 
+For example, to enable `iburst` and set `maxsources` to 2, use the 
+URL `ntp://ntp.example.com?iburst=true&maxsources=2`. 
+The only exception option is NTS, as explained in the next section.
+
+## Use Another NTS Server with NTS Enabled
```
Comment (erinecon) [inline docs/how-to/setup-time-sources.md:22]: I recommend updating this heading to `## Use another NTS server with NTS enabled`

File: docs/how-to/setup-time-sources.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+To use another NTS server with NTS enabled for the time source, follow a 
+similar process as using an NTP server but change the protocol to `nts`.
+For example, use `nts://ntp.example.com`.
```
Comment (erinecon) [inline docs/how-to/setup-time-sources.md:26]: I assume the user is suppose to use `nts://ntp.example.com` in the `sources` charm configuration? I think it would be good to be clearer in this sentence.

File: docs/index.md [hunk 9]
```diff
@@ -1,30 +1,59 @@
...
-for deploying and managing the [Chrony](https://chrony-project.org) NTP server 
+# Chrony Operator
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+for deploying and managing the [Chrony](https://chrony-project.org) NTP server
 in your systems.
 
-This charm simplifies the configuration and maintenance of `chrony` across a 
-range of environments, enabling basic time provision and synchronization with 
+This charm simplifies the configuration and maintenance of `chrony` across a
```
Comment (erinecon) [inline docs/index.md:7]: Nitpick: `chrony` --> Chrony 

Unless there's a reason why you used code formatting here?

File: docs/index.md [hunk 10]
```diff
@@ -1,30 +1,59 @@
...
+
+The Chrony Operator is a member of the Ubuntu family. It's an open-source project that warmly welcomes community 
+projects, contributions, suggestions, fixes, and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
+- [Contribute](https://github.com/canonical/chrony-operator/blob/main/CONTRIBUTING.md)
+
+Thinking about using the <charm-name> Operator for your next project?
```
[the remark this answers is not in the document]
Comment (weiiwang01) [reply docs/index.md]: Updated, thanks!

File: docs/index.md [hunk 11]
```diff
@@ -1,30 +1,59 @@
...
 NTP servers.
 
-## Project and community
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more. 
+For Chrony, this includes: 
+* time source management
+* NTS (Network Time Security) and NTS certificate management
+* Enhanced observability and monitoring
+
+The Chrony charm allows for deployment on many different virtualization platforms, from [lxd](https://canonical.com/lxd) to
```
[the remark this answers is not in the document]
Comment (weiiwang01) [reply docs/index.md]: Updated, thanks!

File: docs/index.md [hunk 12]
```diff
@@ -1,30 +1,59 @@
...
-This charm simplifies the configuration and maintenance of `chrony` across a 
-range of environments, enabling basic time provision and synchronization with 
+This charm simplifies the configuration and maintenance of `chrony` across a
+range of environments, enabling basic time provision and synchronization with
 NTP servers.
 
-## Project and community
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more. 
+For Chrony, this includes: 
+* time source management
```
[the remark this answers is not in the document]
Comment (weiiwang01) [reply docs/index.md]: Updated, thanks!

File: docs/index.md (hunk 9 above)
Comment (weiiwang01) [reply docs/index.md]: Updated to Chrony, thanks!

File: docs/how-to/setup-time-sources.md (hunk 8 above)
Comment (weiiwang01) [reply docs/how-to/setup-time-sources.md:26]: I combined the next sentence about the charm configuration with this one; I hope it's clearer.

File: docs/how-to/setup-time-sources.md (hunk 6 above)
Comment (weiiwang01) [reply docs/how-to/setup-time-sources.md]: Updated alongside other headings, thanks!

File: docs/how-to/setup-time-sources.md (hunk 5 above)
Comment (weiiwang01) [reply docs/how-to/setup-time-sources.md]: Updated, thanks!

File: docs/how-to/setup-time-sources.md (hunk 4 above)
Comment (weiiwang01) [reply docs/how-to/setup-time-sources.md]: Updated, thanks!

File: docs/how-to/enable-nts.md (hunk 3 above)
Comment (weiiwang01) [reply docs/how-to/enable-nts.md:42]: Yes, it should match the `juju add-secret` command output.

File: docs/how-to/enable-nts.md (hunk 2 above)
Comment (weiiwang01) [reply docs/how-to/enable-nts.md]: Okay, I rephrased it a little. What do you think?

File: docs/how-to/enable-nts.md (hunk 1 above)
Comment (weiiwang01) [reply docs/how-to/enable-nts.md]: Updated, thanks!

File: docs/how-to/setup-time-sources.md (hunk 8 above)
Comment (erinecon) [reply docs/how-to/setup-time-sources.md:26]: Yes, I think it's clearer now. Thanks!

**Review by erinecon — DISMISSED** [review]
Thanks for making these doc changes!

File: docs/how-to/enable-nts.md (hunk 3 above)
Comment (erinecon) [reply docs/how-to/enable-nts.md:42]: Thanks for explaining! As a nitpick, I think you should update the `...` to something like `<output from add-secret>`, or maybe something more concise than that. I leave it up to your discretion.

File: docs/how-to/enable-nts.md (hunk 2 above)
Comment (erinecon) [reply docs/how-to/enable-nts.md]: It looks good to me :)

**Review by erinecon — APPROVED** [review]

**Review by amandahla — APPROVED** [review]
